packages/api/src/external/carequality/command/cq-directory/__tests__/search-cq-directory.test.ts

```diff
@@ -0,0 +1,73 @@
+import { faker } from "@faker-js/faker";
+import convert from "convert-units";
+import { makeAddressWithCoordinates } from "../../../../../domain/medical/__tests__/location-address";
+import { makePatient } from "../../../../../domain/medical/__tests__/patient";
+import { CQDirectoryEntry } from "../../../cq-directory";
+import { makeCQDirectoryEntry } from "../../../__tests__/cq-directory";
+import {
+  searchCQDirectoriesAroundPatientAddresses,
+  searchCQDirectoriesByRadius,
+} from "../search-cq-directory";
+
+describe("searchCQDirectoriesAroundPatientAddresses", () => {
+  let searchCQDirectoriesByRadius_mock: jest.Func;
+  beforeAll(() => {
+    jest.restoreAllMocks();
+  });
+  beforeEach(() => {
+    searchCQDirectoriesByRadius_mock = jest.fn(
+      async () => []
+    ) as typeof searchCQDirectoriesByRadius;
+  });
+  afterEach(() => {
+    jest.clearAllMocks();
+  });
+  afterAll(() => {
+    jest.restoreAllMocks();
+  });
+
+  it("returns empty array if no coordinates", async () => {
+    const patient = makePatient();
+    const result = await searchCQDirectoriesAroundPatientAddresses({ patient });
+    expect(result).toBeTruthy();
+    expect(result.length).toEqual(0);
+    expect(searchCQDirectoriesByRadius_mock).not.toHaveBeenCalled();
+  });
+
+  it("returns search result when there are coordinates", async () => {
+    const address = makeAddressWithCoordinates();
+    const patient = makePatient({ data: { address: [address] } });
+    const expectedResult: CQDirectoryEntry[] = [makeCQDirectoryEntry()];
+    searchCQDirectoriesByRadius_mock = jest.fn(() => expectedResult);
+
+    const result = await searchCQDirectoriesAroundPatientAddresses({
+      patient,
+      _searchCQDirectoriesByRadius: searchCQDirectoriesByRadius_mock,
+    });
+    expect(result).toBeTruthy();
+    expect(result.length).toEqual(expectedResult.length);
```

Comment:
in line with the approach you took in the below test, I would remove these two lines since they dont actually test anything in the function 

`expect(result).toBeTruthy();
    expect(result.length).toEqual(expectedResult.length);`

Reply:
I don't think I understand what you mean. Those two lines are important parts of the test: we're checking if the function returns the CQ directory entries returned by `searchCQDirectoriesByRadius`.

Reply:
Your right. I misunderstood the test.